### PR TITLE
Permission to merge to development

### PR DESCRIPTION
**Pull request by peterrobert**
### MILESTONE: PERMISSION TO MERGE TO DEVELOPMENT
**In this milestone I worked on the following**

- [ ] Used fetch to make HTTP requests to an API Endpoint.

- [ ] Used Async 

- [ ] Created a design

### CODE REVIEW

File: src/current.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    const long = position.coords.longitude;
+
+    async function getLocation() {
+      const response = await fetch(
+        `https://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${long}&appid=${Key.theKey()}`,
+        {
+          mode: 'cors',
+        },
+
+      );
```
Comment (lekegitrepo) [inline src/current.js]: - **_It should have error handling for `promises` or `async/await` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/error_handling_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))_**

File: src/searched.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      );
+
+      const weatherData = await response.json();
+
+
+      Logic.logicInfo(weatherData);
+    }
+
+    getLocation();
+  }
```
Comment (lekegitrepo) [inline src/searched.js]: - **_It should have error handling for `promises` or `async/await` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/error_handling_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))_**

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Request changes :recycle:
### Well done :muscle: you have done a good job. However, you need to implement the following :point_down: requirements before this project can be approved.

- **_It should have a toggle control to determine if the result will be shown in Celsius or Fahrenheit *(There are two valid solutions: students can make two different requests for the Celsius data and the Fahrenheit data OR make only one request and use math to convert it to the other unit)* ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))_**

- **_Ensure that students work with Git Flow: production `master` branch + main `develop` branch + Pull Requests from feature branches to the `develop` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/git_flow_requirement.png) from Pathwright)_**

### Optional:

- **_It has a ‘loading’ component that displays from the time the form is submitted until the information comes back from the API ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/loading_component_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))_**

#### _Reviewed by Temitayo Adeleke_

**Comment by peterrobert** [comment]
Fixed all the issues

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @peterrobert 
### You have done well so far :clap: However, there are a few issues to fix:-
- [x] Install all linting dependencies as required configure and run these commands, `npx eslint .` and `npx stylelint "**/*.{css,scss}"` if it thows errors, kindly fix them
- [x] In the scripts in package.json, you can have better scripting, see below
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```
### Make the changes and submit for another code review. Good luck!
- Happy coding!

**Comment by peterrobert** [comment]
Install all linting dependencies as required configure and run these commands, npx eslint . and npx stylelint "**/*.{css,scss}" if it thows errors, kindly fix them 

this is working perfectly kindly recheck

**Review by rOluochKe — APPROVED** [review]
Hello @peterrobert 
### Well done, you got it right :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**